src/Connector.php

```diff
@@ -2,8 +2,8 @@
 
```

Comment:
Might be good to add some more structure to the folders for the following classes. I.e. a folder for `connectors`, `extensions`, `models` etc so its better structured :)